spec/models/outgoing_message_spec.rb

```diff
@@ -161,11 +161,99 @@
 
   describe '#subject' do
```

Comment:
Are these effectively a replacement for the `"when working out follow up subjects"` specs in [/spec/mailers/outgoing_mailer_spec.rb](https://github.com/mysociety/alaveteli/blob/2819-outgoing-message-to-from-subject-2/spec/mailers/outgoing_mailer_spec.rb#L61)? If the functionality's moved we should probably get rid of the `OutgoingMailer` versions.